### ISSUE TITLE
[Techstack-beta] Design: Narrow menu not keyboard accessible

### ISSUE THREAD

**Issue by samueldr** [NixOS/nixos-homepage#1252]
The hamburglar icon element can't be reached via keyboard navigation. It would also fail other navigation "guessing" algorithms.

Going to e.g. the Explore page:

 - On the old version of the site, tabbing once goes to the logo element, then the second tab goes to the hamburglar icon, which can be activated using Enter.
 - On the new version of the site, there is no way to reach the element and thus activate it.
